pom.xml

```diff
@@ -217,7 +214,6 @@
         <version.org.scannotation>1.0.3</version.org.scannotation>
         <version.org.slf4j>1.7.7.jbossorg-1</version.org.slf4j>
         <version.org.syslog4j>0.9.30</version.org.syslog4j>
-        <version.org.testng>6.1.1</version.org.testng>
```

Comment:
we have no tests written in testng, it is a leftover from arquillian container that used to be in wildfly codebase.